src/slices/userSlice.js

```diff
@@ -47,7 +47,7 @@ export const userSlice = createSlice({
     builder
       //Login/Signup
       .addCase(authenticateUser.pending, (state) => {
```

Comment:
isn't builder outdated in RTK
can directly write cases ````[authenticateUser.pending]``` format

Reply:
Yes, but at the time, I was just following the official docs and it used builder, will refactor it later